lib/mailer.rb

```diff
@@ -1,3 +1,5 @@
+require 'erb'
```

Comment:
What is this for?

Reply:
I kept running into an error that ERB was being used without explicitly being required, this fixes it. Might be a ruby version thing, can't see the harm in being cautious